opensafely/codelists.py

```diff
@@ -210,14 +221,16 @@ def check():
         )
 
     try:
-        return check_upstream(codelists_dir)
+        upstream_check = check_upstream(codelists_dir)
     except requests.exceptions.ConnectionError:
         print(
             f"Local codelists OK; could not contact {OPENCODELISTS_BASE_URL} for upstream check,"
             "try again later"
         )
         return True
 
+    return upstream_check
```

Comment:
I think returning `False` here will cause the entire command to exit with an error. See:
https://github.com/opensafely-core/opensafely-cli/blob/b87bef3900a858959bbd19099c770c4730e38668/opensafely/__init__.py#L136-L142

Which I think is not the behaviour you wanted.

Reply:
No, it's not, thanks.